launcher/src/test/java/org/apache/spark/launcher/LauncherServerSuite.java

```diff
@@ -17,19 +17,42 @@
 
 package org.apache.spark.launcher;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
 import java.io.Closeable;
+import java.io.EOFException;
 import java.io.IOException;
+import java.io.InputStream;
+import java.io.ObjectOutputStream;
 import java.net.InetAddress;
 import java.net.Socket;
+import java.util.List;
+import java.util.concurrent.ArrayBlockingQueue;
 import java.util.concurrent.BlockingQueue;
 import java.util.concurrent.LinkedBlockingQueue;
 import java.util.concurrent.Semaphore;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
 
+import org.apache.spark.launcher.LauncherProtocol.Hello;
+import org.apache.spark.launcher.LauncherProtocol.Message;
+import org.apache.spark.launcher.LauncherProtocol.SetAppId;
+import org.apache.spark.launcher.LauncherProtocol.SetState;
+import org.apache.spark.launcher.LauncherProtocol.Stop;
+import org.apache.spark.launcher.SparkAppHandle.State;
 import org.junit.Test;
-import static org.junit.Assert.*;
+import org.mockito.Mockito;
+
+import com.google.common.collect.Lists;
```

Comment:
This module doesn't depend on Guava, so this would probably cause a failure (at least when using maven). With Java 7 it's pretty easy to avoid these particular Guava classes, though.